src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/osds.po.ts

```diff
@@ -14,6 +14,7 @@ export class OSDsPageHelper extends PageHelper {
   };
 
   create(deviceType: 'hdd' | 'ssd', hostname?: string, expandCluster = false) {
+    cy.get('[aria-label="toggle advanced mode"]').click();
```

Comment:
I wanted to introduce it but it looks kind of complicated. Because this will deploy OSDs on all the available HDDs and that's not what we want for testing. We need one host to not have OSD deployed to succesfully test the drain operation. Hence I thought I'll take care of the BDD in a later PR.

Reply:
Ok